patches/minecraft/net/minecraft/block/BlockBed.java.patch

```diff
@@ -0,0 +1,13 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockBed.java
++++ ../src-work/minecraft/net/minecraft/block/BlockBed.java
+@@ -329,4 +329,10 @@
+             return this.field_177036_c;
+         }
+     }
++
++    /* ======================================== FORGE START =====================================*/
++    public boolean rotateBlock(World world, BlockPos pos, EnumFacing axis)
```

Comment:
Rather then expand the number of patches we have, it's preferrible to add this type of stuff to the base Block class.